docs/build.md

```diff
@@ -12,38 +12,70 @@ sudo snap install go --classic
 On other systems or in docker, use the directions here: https://go.dev/doc/install.
 Summary for X86-64 Linux (update GO_VERSION below to the latest stable release):
 ```bash
-GO_VERSION=1.20.3
+GO_VERSION=1.20.5
 wget "https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz"
 rm -rf /usr/local/go && tar -C /usr/local -xzf go${GO_VERSION}.linux-amd64.tar.gz
 echo "export PATH=$PATH:/usr/local/go/bin" >> .profile
 source .profile
 ```
 
-2. Clone the repo:
+### 2. Clone the repo
 ```bash
 git clone https://github.com/athanorlabs/atomic-swap.git
 cd atomic-swap
+
+# Optional: Check out the exact source code of the latest release
+git checkout "$(git describe --abbrev=0 --tags)"
 ```
 
-3. Finally, build the repo:
+### 3. Finally, build the repo
+
+#### Option 1: Build the latest released/tagged version.
 ```bash
-make build
+make build-release
 ```
+This option, and the ones below, all create `swapd` and `swapcli` binaries in
+a `bin` subfolder.
 
-This creates `swapd` and `swapcli` binaries in the `bin` folder.
+#### Option 2: Build release binaries without checking out the repo
 
-Note: if you wish to run a bootnode (see [here](./bootnode.md)), run `make build-all`.
+The make-target in "Option 1" uses "go install", a command that can be run
+without checking out the source code. If you just want the release binaries in a
+subfolder named "bin", you can use:
+```bash
+GOBIN=${PWD}/bin go install -tags=prod github.com/athanorlabs/atomic-swap/cmd/...@latest
```

Comment:
isn't this the exact command of `build-release`? this seems unneeded

Reply:
It is, but these are instructions on how to install the binaries without checking out the repository, so they won't have access to the Makefile.

Reply:
maybe this should be option 1 then, before the cloning the repo option

Reply:
Done, but that required a major rewrite of the top-level headings in `build.md`. The links in the description will be more helpful than the diffs at this point.